components/backup-stream/src/endpoint.rs

```diff
@@ -564,6 +545,26 @@ where
         Ok(())
     }
 
+    /// initialize a range: it simply scan the regions with leader role and send
+    /// them to [`initialize_region`].
+    pub fn initialize_range(&self, start_key: Vec<u8>, end_key: Vec<u8>) -> Result<()> {
+        let mut pager = RegionPager::scan_from(self.regions.clone(), start_key, end_key);
+        loop {
+            let regions = pager.next_page(8)?;
```

Comment:
Please add some comments about the magic number.

Reply:
In fact I think this can be removed...

Reply:
(Perhaps we'd better to keep it -- or once there are too many regions we may consume too many memory here.